scripts/buildJsr.ts

```diff
@@ -0,0 +1,17 @@
+import process from "node:process";
+import fs from "node:fs";
+import {getPackageJson} from "./utils.ts";
+
+const {name, version} = await getPackageJson();
+
+fs.writeFileSync(
+  `${process.cwd()}/deno.json`,
```

Comment:
I'm a bit confused.  What is this file for?   Only to support Deno?  Does jsr work with the other runtimes?

Reply:
`jsr.json` is a subset of `deno.json`. All the properties of JSR can bet set in the `deno.json`. Similarly the JSR commands e.g. `npx jsr publish` can also be replaced via `deno publish`. 

JSR is default and builtin package manger for Deno, but also published as independent package for community moving from NodeJS to JSR registry.